src/LaxZonedDateTimes.jl

```diff
@@ -39,6 +40,11 @@ function LaxZonedDateTime()
     LaxZonedDateTime(DateTime(0), utc, utc, false)
 end
 
+function LaxZonedDateTime(utcdt::UTCDateTime)
+    utc = TimeZone("UTC")
```

Comment:
```suggestion
    utc = tz"UTC"
```
I thought this was more efficient since it happens at macroexpand time and thus gets precompiled? I've been doing it that way forever but it looks like `TimeZone("UTC")` is used a few places in this package.